smqtk_descriptors/interfaces/descriptor_generator.py

```diff
@@ -191,17 +191,25 @@ def iter_tocompute_data() -> Generator[DataElement, None, None]:
 
             end_of_iter[0] = last_i
 
-        descr_vec_iter = self.generate_arrays(iter_tocompute_data())
+        def iter_tocompute_data() -> Generator[DataElement, None, None]:
```

Comment:
Oh my, I'm sorry, my last comment was wrong. This is an internal function, thus merely an implementation detail and not exposed outside of this context. This deprecation shim is not necessary.